app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,87 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$parchment: #EBE5D5;
+$dark-blue: #112230;
+$light-blue: #68B7E1;
+$ocean-green: #70A58B;
+$dark-ocean_green: #649d81;
+$grey: #3D4B50;
+$sand: #B9A167;
+$crab-red: #E24D34;
+$shrimp-pink: #E49E78;
+$dark-shrimp_pink: #E19268;
+$clam-tan: #D7CAB7;
+$dark-clam_tan: #C2AE91;
```

Comment:
Color `#C2AE91` should be written as `#c2ae91`<br>Name of variable `dark-clam_tan` should be written in all lowercase letters with hyphens instead of underscores